solr/CHANGES.txt

```diff
@@ -4,6 +4,14 @@ This file lists Solr's raw release notes with details of every change to Solr.
 Most people will find the solr-upgrade-notes.adoc file more approachable.
 https://github.com/apache/lucene-solr/blob/master/solr/solr-ref-guide/src/solr-upgrade-notes.adoc
 
+==================  8.8.1 ==================
```

Comment:
Does it make sense to add upgrade notes and mention the property that'll allow them to upgrade the server without upgrading the client?

Reply:
Yes, I very much think so. Similar to the fix for the default autoscaling policy in 8.6.1

Reply:
The property already defaults to true, which means the `base_url` will be stored again, so really there's nothing people need to do with the sys prop to get an older SolrJ to work with an 8.8.1 server again. In reality, the sys prop is only there for people that are running the latest SolrJ and want to prepare their clusters for 9.x by explicitly setting to `false`. Doubt anyone will actually do that so the sys prop is probably not all that useful.

I still need to test to make sure an upgrade of server nodes from 8.8.0 to 8.8.1 does restore the `base_url` in ZK.

Reply:
Added a note there, let me know if that makes sense?